src/wallet/rpcwallet.cpp

```diff
@@ -3531,6 +3532,9 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
     if (contextualTx.nVersion == 1) {
         contextualTx.nVersion = 2; // Tx format should support vjoinsplits 
     }
+    if (NetworkUpgradeActive(chainActive.Height(), Params().GetConsensus(), Consensus::UPGRADE_OVERWINTER)) {
+        contextualTx.nExpiryHeight = chainActive.Height() + blockExpiry;
```

Comment:
Yeah I considered that, but there was a comment in there about not including expiryHeight there, actually. 
Line 5866 `src/main.cpp`:    
 `// Expiry height is not set. Only fields required for a parser to treat as a valid Overwinter V3 tx.`
Think @bitcartel wrote that?

Reply:
Is that a declarative or imperative comment?

Reply:
I wrote that comment because expiry height does not need to be set to make a valid v3 overwinter tx.  CreateNewContextualCMutableTransaction() only checks to see if Overwinter is active or not based on an input height, and if active, the mutable tx returned is a v3.  You can fill in the expiry height at the call site, where you have a better idea of the context, e.g. whether its going to use the chain tip or some other value.

Reply:
My main concern is that this needs to catch every usage, and we have already been through that process adding in ` CreateNewContextualCMutableTransaction` call sites. Please go through and set a default expiry height at every call site individually.

Reply:
Went through and checked each call site, added to `createrawtransaction`